pom.xml

```diff
@@ -177,6 +177,8 @@
         <dep.casandra.version>4.17.0</dep.casandra.version>
         <dep.minio.version>8.5.6</dep.minio.version>
         <dep.iceberg.version>1.3.1</dep.iceberg.version>
+        <!-- Nessie version (matching to Iceberg release) must be bumped along with Iceberg version bump to avoid compatibility issues -->
+        <dep.nessie.version>0.59.0</dep.nessie.version>
```

Comment:
This is needed by Iceberg only, so let's keep it in trino-iceberg/pom.xml

Reply:
This is just below iceberg version so seems consistent

Reply:
Ping @findepi

Reply:
it still can be used in properties file, just not in `trino-testing-services`.

Reply:
Which will make this change more complicated